content/en/roadmap.md

```diff
@@ -9,79 +9,124 @@ type: page
-  - [x] [Verify Helm charts with cosign](https://github.com/fluxcd/source-controller/issues/914)
+- [x] Conformance testing
+  - [x] End-to-end testing for bootstrap on AMD64 and ARM64 clusters
+  - [x] End-to-end testing for Flux self-upgrade
+  - [x] End-to-end testing for multi-tenancy lockdown
 
-- **API consolidation** ([fluxcd/flux2#1601](https://github.com/fluxcd/flux2/issues/1601))
-  - [ ] Adopt Kubernetes [kstatus](https://github.com/kubernetes-sigs/cli-utils/tree/v0.25.0/pkg/kstatus#conditions) standard conditions
-  - [ ] Standardize events and status conditions metadata
-  - [ ] [Standardize the OpenAPI validation](https://github.com/fluxcd/flux2/issues/2993) for the `toolkit.fluxcd.io` CRDs
+- [ ] Documentation
+  - [x] Install and bootstrap guides for Kubernetes (Generic Git, GitHub, GitLab, BitBucket)
+  - [x] Install and bootstrap guides for managed Kubernetes (AWS, Azure, Google Cloud)
+  - [x] `gitrepositories.source.toolkit.fluxcd.io` API specification
+  - [ ] `kustomizations.kustomize.toolkit.fluxcd.io` API specification
+    [kustomize-controller#664](https://github.com/fluxcd/kustomize-controller/issues/664)
 
-- **Git improvements** ([fluxcd/flux2#3039](https://github.com/fluxcd/flux2/issues/3039))
-  - [ ] [Consolidate Git implementations](https://github.com/fluxcd/pkg/issues/245)
-  - [ ] [End-to-end testing for Git protocols](https://github.com/fluxcd/pkg/issues/334)
+### Flux Helm GA
+
+The goal of this milestone is to make a general available release for the Flux Helm APIs
+and the Flux Helm functionalities.
+
+The completion of this milestone will be marked by the `v2.1.0` release of the Flux distribution and CLI.
+
+- [ ] API promotions to GA
+  - [ ] `helmrepositories.source.toolkit.fluxcd.io/v1`
+  - [ ] `helmcharts.source.toolkit.fluxcd.io/v1`
+  - [ ] `helmreleases.helm.toolkit.fluxcd.io/v1`
+
+- [ ] Reconcilers
+  - [x] [OCI support and Cosgin verification for HelmChart](https://github.com/fluxcd/flux2/tree/main/rfcs/0002-helm-oci#implementation-history)
+  - [ ] [Atomic reconciliation of HelmReleases](https://github.com/fluxcd/helm-controller/pull/532)
+  - [ ] [Standardize events and status conditions for HelmReleases](https://github.com/fluxcd/helm-controller/issues/487)
+
+- [ ] CLI
+  - [ ] [Cover more Helm release configuration options](https://github.com/fluxcd/flux2/issues/213)
+
+- [ ] Documentation
+  - [x] `helmrepositories.source.toolkit.fluxcd.io` API specification
+  - [x] `helmcharts.source.toolkit.fluxcd.io` API specification
+  - [ ] `helmreleases.helm.toolkit.fluxcd.io` API specification
+
+### Flux Notifications GA (TBA)
+
+- [ ] API promotions to GA
+  - [ ] `alerts.notification.toolkit.fluxcd.io/v1`
+  - [ ] `providers.notification.toolkit.fluxcd.io/v1`
+
+This milestone's tasks haven't been determined yet.
+
+### Flux Image Automation GA (TBA)
+
+- [ ] API promotions to GA
+  - [ ] `imagepolicies.image.toolkit.fluxcd.io/v1`
+  - [ ] `imageupdateautomations.image.toolkit.fluxcd.io/v1`
```

Comment:
Does this intentionally excludes ImageRepository with the plan to merge it with OCIRepository or something idea?

Reply:
Yes, since we haven't decided how to move forward with ImageRegistry I left it out for now.